pypeit/calibrations.py

```diff
@@ -414,15 +414,15 @@ def get_bias(self):
         # Commenting this out for now until it gets fixed. I'm not convinced this should be here.
 
         # If we need to make a bad pixel mask using the bias frames, do it now
-        #if self.par['badpix']:
-        #    # Instantiate the shape here, based on the shape of the bias image
-        #    self.shape = self.msbias.shape
-        #
-        #   # Build it
-        #    self.msbpm = self.spectrograph.bpm(shape=self.shape, det=self.det, msbias=self.msbias)
-        #
-        #    # Record it
-        #    self._update_cache('bpm', 'bpm', self.msbpm)
+        if self.par['badpix']:
+            # Instantiate the shape here, based on the shape of the bias image
+            self.shape = self.msbias.shape
+
+            # Build it
+            self.msbpm = self.spectrograph.bpm(shape=self.shape, det=self.det, msbias=self.msbias)
```

Comment:
The parent call takes `(filename, det, shape=)` so I think this will crash for all spectrographs except WHT.  I presume that is not as intended.

Reply:
If we allow `msbias=` to become an option, then we should either add **kwargs or `msbias=` throughout.